apps/passport-server/test/generic-issuance/util.ts

```diff
@@ -152,15 +188,20 @@ export async function requestTicketsFromPipeline(
   /**
    * Is owned by this identity.
    */
-  identity: Identity
+  identity: Identity,
+  /**
+   * Whether to include the v4 ID in the EmailPCD.
+   */
+  includeV4Id: boolean = false
```

Comment:
this function is used to generate credentials for tests, and this flag lets the caller choose whether the email pcd included in the credential does or doesn't have the requestor's v4 semaphore commitment in it